contracts/queue/src/contract.rs

```diff
@@ -167,11 +175,25 @@ fn query_reducer<S: Storage, A: Api, Q: Querier>(
     Ok(ReducerResponse { counters: out })
 }
 
+fn query_list<S: Storage, A: Api, Q: Querier>(deps: &Extern<S, A, Q>) -> StdResult<ListResponse> {
+    Ok(ListResponse {
+        swaps: all_swap_ids(&deps.storage)?,
```

Comment:
If you're gonna keep these tests, better to rename these to a generic name like, `empty_range`, something like that.

Reply:
Cleaned that up and improved them a bit